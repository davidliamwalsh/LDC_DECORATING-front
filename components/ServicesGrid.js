import { Component } from 'react'
import Link from 'next/link'

class ServicesGrid extends Component {
  render () {
    return (
      <div className="c-service-sector--grid">
        <div className="c-service-sector--grid__item" data-aos="fade-right">
          <div className="c-service-sector--grid__item--title">
            <h2>Decorative Wall Coverings</h2>
            <hr className="o-line__testimonial" />
          </div>
          <Link href='/services'>
            <a><div className="c-service-sector--grid__item--image c-service-sector--grid__item--wall-coverings" /></a>
          </Link>
          
          <div className="c-service-sector--grid__item--footer">
            <h4>Visual Wall Decorations</h4>
          </div>
        </div>
        <div className="c-service-sector--grid__item" data-aos="fade-left">
          <div className="c-service-sector--grid__item--title">
            <h2>Decorative Wall Coverings</h2>
            <hr className="o-line__testimonial" />
          </div>
          <Link href='/services'>
            <a><div className="c-service-sector--grid__item--image c-service-sector--grid__item--wall-coverings" /></a>
          </Link>
          
          <div className="c-service-sector--grid__item--footer">
            <h4>Visual Wall Decorations</h4>
          </div>
        </div>
        <div className="c-service-sector--grid__item" data-aos="fade-right">
          <div className="c-service-sector--grid__item--title">
            <h2>Decorative Wall Coverings</h2>
            <hr className="o-line__testimonial" />
          </div>
          <Link href='/services'>
            <a><div className="c-service-sector--grid__item--image c-service-sector--grid__item--wall-coverings" /></a>
          </Link>
          
          <div className="c-service-sector--grid__item--footer">
            <h4>Visual Wall Decorations</h4>
          </div>
        </div>
        <div className="c-service-sector--grid__item" data-aos="fade-left">
          <div className="c-service-sector--grid__item--title">
            <h2>Decorative Wall Coverings</h2>
            <hr className="o-line__testimonial" />
          </div>
          <Link href='/services'>
            <a><div className="c-service-sector--grid__item--image c-service-sector--grid__item--wall-coverings" /></a>
          </Link>
          
          <div className="c-service-sector--grid__item--footer">
            <h4>Visual Wall Decorations</h4>
          </div>
        </div>
        <div className="c-service-sector--grid__item" data-aos="fade-right">
          <div className="c-service-sector--grid__item--title">
            <h2>Decorative Wall Coverings</h2>
            <hr className="o-line__testimonial" />
          </div>
          <Link href='/services'>
            <a><div className="c-service-sector--grid__item--image c-service-sector--grid__item--wall-coverings" /></a>
          </Link>
          
          <div className="c-service-sector--grid__item--footer">
            <h4>Visual Wall Decorations</h4>
          </div>
        </div>
        <div className="c-service-sector--grid__item" data-aos="fade-left">
          <div className="c-service-sector--grid__item--title">
            <h2>Decorative Wall Coverings</h2>
            <hr className="o-line__testimonial" />
          </div>
          <Link href='/services'>
            <a><div className="c-service-sector--grid__item--image c-service-sector--grid__item--wall-coverings" /></a>
          </Link>
          
          <div className="c-service-sector--grid__item--footer">
            <h4>Visual Wall Decorations</h4>
          </div>
        </div>
      </div>
    )
  }
}

export default ServicesGrid