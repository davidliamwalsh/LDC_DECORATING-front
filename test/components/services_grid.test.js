import ServicesGrid from '../../components/ServicesGrid'

describe('<ServicesGrid />', () => {
  it('it renders correctly', () => {
    const component = shallow(
      <ServicesGrid />
    )

    expect(component).toMatchSnapshot()
  })
})