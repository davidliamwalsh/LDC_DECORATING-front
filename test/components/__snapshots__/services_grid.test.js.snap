// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ServicesGrid /> it renders correctly 1`] = `ShallowWrapper {}`;
